user-guide/Troubleshooting/Procedures/TroubleshootingSLScriptingOutOfMemoryException.md

```diff
@@ -0,0 +1,54 @@
+---
+uid: TroubleshootingSLScriptingOutOfMemoryException
+---
+
+# Investigating OutOfMemoryException occurrence
+
+> [!IMPORTANT]
+> This section might include some information that is only applicable to Skyline employees and/or links that are only accessible to Skyline employees.
+
+In case you experience an SLScripting crash because the process ran out of memory or you observe a memory leak to be present in the SLScripting process, analyzing a memory dump of the SLScripting process can provide more info on what is causing the leak.
+
+Note that by default a single instance of the SLScripting process is spun up by DataMiner. This process is responsible for executing the QActions of protocols. QActions execute C# code which is executed by the CLR of the .NET Framework. Therefore, if you observe a memory leak in SLScripting, a good stating point is to analyze the managed memory of the SLScripting process. Note however that in some situations a leak could be present in unmanaged memory, for example when unmanaged resources are not cleaned up correctly.
```

Comment:
Maybe you could move the note about the single instance to a [!TIP] banner that links to [Configuring a separate SLScripting](https://docs.dataminer.services/user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Configuration_of_DataMiner_processes.html#configuring-a-separate-slscripting-process-for-each-slprotocol-process) and [Element in Protocol logging](https://docs.dataminer.services/user-guide/Troubleshooting/Logging/Consulting_the_DataMiner_logs/DataMiner_logging/Element_in_Protocol_logging.html?q=slelementin).
This could be placed right after the section that talks about taking a dump, as this would be the first place where you would need to take this in consideration

Reply:
added links and tip